css-three-column-layout/css-three-column-layout/styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  background: #039be5;
+  color: white;
+  box-sizing: border-box;
+  min-width: 0;
+  min-height: 0;
+}
+
+body {
+  margin: auto;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+header {
+  padding: 30px;
+  text-align: center;
+}
+
+img {
+  width: 82px;
+  height: 85px;
+  background-color: white;
+  align-content: center;
+}
+
+h3 {
+  font-size: 23px;
+  width: 300px;
+}
+
+p {
+  color: gainsboro;
+  width: 320px;
+  font-size: 18px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row-1 {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  width: 1200px;
+}
+
+.row-2 {
+  display: flex;
+  justify-content: center;
+  width: 1100px;
+  gap: 8.5%;
+  margin: auto;
+}
+
+.column-full {
+  width: 1200px;
+}
+
+.column1 {
```

Comment:
the column classes should have a percentage width, and since all of your columns are the same width, you should only need one column class, something like `column-one-third`. You also want to avoid making multiple classes with the same css. Better to use the same one everywhere you want the same CSS.

Reply:
Thank you Brett! I really want to make sure I have this nailed down, and the practice is really helping me.